src/components/Icons/index.tsx

```diff
@@ -10,21 +10,21 @@ interface IProps {
   color?: string;
   size?: string;
   classPrefix?: string;
-  tag?: string;
+  tag?: 'i';
```

Comment:
i think tag name should be a string here. 'i' is just the default value, not enum